### DISCUSSION TITLE
Is there a callback that tells me which day I landed on?

### DISCUSSION THREAD

**Discussion by tdserviceco** [wojtekmaj/react-calendar#587] [Q&A]
Hi! using your React-Calendar to my exam project and it does ALMOST everything in the checkbox I need.
but this is a nitpick.. Can I when I load-in the React-Calendar get some kind of trigger on the current day i'm landing on? Example once i load the calender I will land on 2022-01-27 and trigger the onClickDay.
Sorry if this sound weird.

Have a good day.

**Comment by wojtekmaj** [comment]
The way React-Calendar calculates which month/year/... to show is very complicated but in a simplified way it can be described like so:

If you provided activeStartDate and view, it will explicitly show the month/year/... you requested.

If not, and if you provided value, it will show the month that contains the value.

If not, it will show the month that contains today's date.

Which I guess in your case is all you need to know. Without value provided, you will see the calendar opened on current month by default.
